fluent-plugin-google-cloud.gemspec

```diff
@@ -21,9 +21,9 @@ eos
 
   gem.add_runtime_dependency 'fluentd', '~> 0.10'
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
-  gem.add_runtime_dependency 'google-api-client', '> 0.9'
+  gem.add_runtime_dependency 'google-api-client', '> 0.9', '< 0.10'
```

Comment:
I think there is a separate issue with `google-api-client` (which was released on Feb 13).